.github/workflows/ci-actions.yml

```diff
@@ -452,9 +512,9 @@ jobs:
         shell: bash
         run: tar -xzf maven-repo.tgz -C ~
       - name: Build with Maven
-        run: mvn -B --settings .github/mvn-settings.xml -Dno-native -Dno-format -DskipTests -Dtcks install
+        run: mvn -B --settings .github/mvn-settings.xml -Dno-native -Dno-format -DskipTests -Dtcks install ${{ needs.build-jdk11.outputs.gib_args }}
       - name: Verify with Maven
-        run: mvn -B --settings .github/mvn-settings.xml -f tcks/pom.xml verify
+        run: mvn -B --settings .github/mvn-settings.xml -Dtcks -pl tcks -amd verify ${{ needs.build-jdk11.outputs.gib_args }}
```

Comment:
While `-f` is more efficient, here with we need `-pl` we need the entire reactor (=all modules) so that GIB can properly detect changes and map them to modules.
With `-f` GIB would only "see" a fraction of the modules and change detection would not work as we need it to.

Reply:
Won't this thing run the build and the tests of all the modules triggered by amd?

Reply:
No, because the profile overwrites the modules so that only `tcks` is left:
```xml
        <profile>
            <id>tcks</id>
            <activation>
                <property>
                    <name>tcks</name>
                </property>
            </activation>
            <modules>
                <module>tcks</module>
            </modules>
        </profile>
```

Reply:
Hmmm, I do that in HV and it adds the modules, it doesn't overwrite them.

Reply:
You are absolutely right!
The thing here is that `-pl tcks -amd` builds `Quarkus - TCK - Parent` and everything that depends on it, which is basically just all those tcks submodules.

Reply:
...and from those modules GIB is removing the ones that are not changed (or don't depend on changed modules).

Example:
```
$ mvn -Dgib -Dgib.dbc -Dtcks -pl tcks -amd validate -Dgib.dsph
[INFO] Scanning for projects...
[INFO] gitflow-incremental-builder 3.12.0 starting...
[INFO] ------------------------------------------------------------------------
[INFO] Changed Artifacts:
[INFO]
[INFO] quarkus-vertx (but deselected)
[INFO]
[INFO] gitflow-incremental-builder exiting...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] Quarkus - TCK - MicroProfile Health                                [jar]
[INFO] Quarkus - TCK - MicroProfile JWT                                   [jar]
[INFO] Quarkus - TCK - MicroProfile Reactive Messaging                    [jar]
[INFO] Quarkus - TCK - MicroProfile OpenAPI                               [jar]
```
PS: I'm lazy 😉; `dbc` is `disableBranchComparison` and `dsph` is `disableSelectedProjectsHandling`.

Reply:
This should also be resolved by now.